src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -233,5 +233,24 @@ internal string CreateSearchQuery(string[] fields, string SearchTerm)
             }
             return booleanQuery.ToString();
         }
+
+        internal void DisposeWriter()
+        {
+            //We need to check if we are not running Dynamo tests because otherwise parallel test start to fail when trying to write in the same Lucene directory location
+            if (!DynamoModel.IsTestMode)
+            {
+                writer?.Dispose();
+                writer = null;
+            }
+        }
+
+        internal void CommitWriterChanges()
+        {
+            if (!DynamoModel.IsTestMode)
+            {
+                //Commit the packages node info indexed
+                writer?.Commit();
```

Comment:
remove package node word here since the info could be about anything

Reply:
comment fixed in commit: https://github.com/DynamoDS/Dynamo/pull/14120/commits/c52a9a6efc21f7d0b6de1894158202c8d6a52b0a